lib/ansible/modules/windows/win_domain_user.ps1

```diff
@@ -164,6 +167,45 @@ If ($state -eq 'present') {
             }
         }
 
+        # Set additional attributes
+        $set_args = @{}
```

Comment:
I would recommend adding `$run_change = $false` before starting the loop, may not affect the Ansible executable but it would potentially cause issues if you are debugging this locally.

Reply:
Sure...

Reply:
Done